packages/api/src/libs/zrx.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import assert from "assert";
+import Client from "./zrx";
+
+const baseUrl = "https://api.0x.org/";
```

Comment:
How do you feel about having real external dependencies, such as the `api.0x.org` server, in our unit tests that are run in CI? Is there a way to mock this?

Reply:
yea this is awkward. i've named these tests "e2e" to denote they should not be run in ci, and `yarn test` will not run this test.  you have to run `yarn e2e` in order for this test to run.  not ideal and open to suggestions here

Reply:
I like the setup you have @daywiss